tests/mock-workflows/app.py

```diff
@@ -0,0 +1,8 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/api/v1/workflow/generate_artifact', methods=['POST'])
+def generate_artifact():
+    return '', 201
```

Comment:
I think it could be useful to be able to mock the response of this end-point as well.

Reply:
Done!